### PR TITLE
Update api requests

### PR DESCRIPTION
**Pull request by jpc20**
# Pull request checklist

## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Current behavior

- The changes to the API broke our current version of the app.

## New behavior

- Updated API requests to reflect the changes on the BE.

### CODE REVIEW

File: src/components/CampaignDetails.vue [hunk 1]
```diff
@@ -60,10 +60,13 @@ export default {
       })
       if (result && (web3.utils.toWei(this.contribution, "ether") > this.campaign.min_contribution.toString())) {
         try {
-          await axios.put(
+          const resp = await axios.put(
             `${VUE_APP_API_URL}campaigns/${this.campaign.id}`,
-            { contributors: (this.campaign.contributors + 1) }
+            {
+              value: (this.$store.getters.getSingleCampaign(this.address).value + this.contribution)
```
Comment (jpc20) [inline src/components/CampaignDetails.vue]: It there a way I can access this campaign without calling this method?

File: src/store/index.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ export default createStore({
         from: manager
       });
       if (result) {
-        const newRequest = { ...request, value: weiValue };
+        const newRequest = { ...request, value: value };
```
Comment (jpc20) [inline src/store/index.js:64]: BE is storing ETH as value.

File: src/components/CampaignDetails.vue [hunk 3]
```diff
@@ -62,7 +62,9 @@ export default {
         try {
           await axios.put(
             `${VUE_APP_API_URL}campaigns/${this.campaign.id}`,
-            { contributors: (this.campaign.contributors + 1) }
+            {
+              value: (this.campaign.value + this.contribution)
```
Comment (jpc20) [inline src/components/CampaignDetails.vue]: This actually doesn't seem to be working... any ideas?

File: src/components/CampaignDetails.vue [hunk 4]
```diff
@@ -58,21 +58,32 @@ export default {
...
+        if (this.contribution > this.campaign.min_contribution) {
+          try {
+            const resp = await axios.post(
+              `${VUE_APP_API_URL}campaigns/${this.campaign.address}/contributor/${this.$store.state.accountNum}`,
+              {
+                address: this.$store.state.accountNum,
+                email: null
+              }
+            );
+          } catch (error) {
```
Comment (jpc20) [inline src/components/CampaignDetails.vue:81]: This is all super ugly and should be refactored if we have time, but it seems to be working!

Comment (edmdc) [reply src/components/CampaignDetails.vue:81]: I think once we start moving some of this logic away of the component and into the store some of this code will be simplified.

**Review by edmdc — APPROVED** [review]

**Pull request merged**